src/nvim/os/win_defs.h

```diff
@@ -3,6 +3,8 @@
 
 #include <windows.h>
 
+#include <uv.h>
```

Comment:
> If we are fine with anything in os/* including uv.h then we can drop the uv.h include.

Yes, though I would frame it as "moving towards our general goal of [include-what-you-use](https://github.com/neovim/neovim/issues/549)". `fs_defs.h` includes uv.h so I guess we are good.